src/extension/options-page/DashboardDialogs/Persona.tsx

```diff
@@ -112,6 +116,14 @@ export function PersonaDeleteDialog(props: PersonaDeleteDialogProps) {
 
     const deletePersona = () => {
         Services.Identity.deletePersona(persona.identifier, 'delete even with private').then(onConfirm)
+        Object.keys(currentImmersiveSetupStatus).forEach(network => {
```

Comment:
It's a bug fix that prevents immersive guide working property if persona was deleted.